pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -1358,14 +1358,14 @@ ORDER BY oid
 oid     typname        typndims  typcollation  typdefaultbin  typdefault  typacl
 16      bool           0         0             NULL           NULL        NULL
 17      bytea          0         0             NULL           NULL        NULL
-18      char           0         3903121477    NULL           NULL        NULL
-19      name           0         3903121477    NULL           NULL        NULL
+18      char           0         3403232968    NULL           NULL        NULL
+19      name           0         3403232968    NULL           NULL        NULL
```

Comment:
`name` has a collation of `C` in PG

???

Reply:
yeah i think that makes sense as name is an internal thing isn't it? either way we don't have C, so i think we can leave this as is